bpf/dns_tracker.h

```diff
@@ -92,7 +92,7 @@ static __always_inline int track_dns_packet(struct __sk_buff *skb, pkt_info *pkt
         if ((flags & DNS_QR_FLAG) == 0) { /* dns query */
             fill_dns_id(pkt->id, &dns_req, dns_id, false);
             if (bpf_map_lookup_elem(&dns_flows, &dns_req) == NULL) {
-                bpf_map_update_elem(&dns_flows, &dns_req, &ts, BPF_ANY);
+                bpf_map_update_elem(&dns_flows, &dns_req, &ts, BPF_NOEXIST);
```

Comment:
You can skip the lookup before this update when using BPF_NOEXIST :)

Reply:
done